zerver/lib/slack_message_conversion.py

```diff
@@ -8,9 +8,10 @@
 # Slack link can be in the format <http://www.foo.com|www.foo.com> and <http://foo.com/>
 LINK_REGEX = r"""
               (<)                                                     # match '>'
-              (http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?
-                  ([a-z0-9]+([\-\.]{1}[a-z0-9]+)*)(\.)                # subdomain
-                  ([a-z]{3,63}(:[0-9]{1,5})?)(\/[^>]*)?               # domain name has chars in range {3,63}
+              (http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?  # protocall and subdomain
```

Comment:
*protocol and www subdomain

Reply:
I just tested this, that Slack also linkify `ftp://`, e.g. ftp://speedtest.tele2.net/.

Reply:
@rht as you mentioned we don't have it in zulip.I'll still map it. I also have to cover the mailto mapping

Reply:
Slack also linkifies `skype:a@b.com`.